sdk/monitor/monitor-opentelemetry-exporter/src/utils/common.ts

```diff
@@ -254,3 +254,7 @@ export function createResourceMetricEnvelope(
   }
   return;
 }
+
+export function shouldCreateResourceMetric(): boolean {
+  return !(process.env.ENV_OPENTELEMETRY_RESOURCE_METRIC_DISABLED?.toLowerCase() === "true");
```

Comment:
Do you want to check for env variable every time export is executed or only check once when SDK is initialized?

Reply:
I'll update this to save the result of the env var check so we aren't reading an env var each time.

Reply:
Updated such that this will only be calculated upon new exporter creation.